### PR TITLE
updates to labs 2-3-5

### PR DESCRIPTION
**Pull request by jkomg**
Edit readme.md for errors and coding issues;.

### CODE REVIEW

File: Lab 3/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Run the following to begin this lab:
 
 
 5. Change current directory to `Stage3/watson-talk`
```
Comment (stevemar) [inline Lab 3/README.md]: i think this text needs to be updated?

**Review by stevemar — APPROVED** [review]
looks fine to me, just one question

**Review by loafyloaf — APPROVED** [review]
lgtm

**Review by stevemar — APPROVED** [review]
LGTM

**Pull request merged**